roles/wazuh/ansible-wazuh-manager/tasks/RedHat.yml

```diff
@@ -16,15 +16,15 @@
     description: NodeJS-$releasever
     baseurl: https://rpm.nodesource.com/pub_6.x/fc/$releasever/x86_64
     gpgkey: https://rpm.nodesource.com/pub/el/NODESOURCE-GPG-SIGNING-KEY-EL
-    gpgcheck: yes
+    gpgcheck: true
   when: ansible_distribution == 'Fedora'
 
 - name: AmazonLinux | Get Nodejs
   shell: |
     set -o pipefail
     curl --silent --location https://rpm.nodesource.com/setup_8.x | bash -
   args:
-    warn: no
+    warn: fakse
```

Comment:
Hi @dj-wasabi , I've noticed that there's a typo here.

Reply:
Thanks, good catch! Pushed the fix.